Added hello-devwahala.txt

devwahala opened the pull request:
Just following through on [Matt's article](https://mattstauffer.com/blog/how-to-contribute-to-an-open-source-github-project-using-your-own-fork/).

typpo commented:
Nice job for your first time.  The build is probably failing because you forked an old version of the repo.  Maybe if you synced master (`git checkout master` and `git pull`), merged it into your branch (`git checkout hello-devwahala` and `git merge master`), and updated this pull request (`git push`), the build will pass.

devwahala commented:
It seems I created a development branch that no longer existed on the repo.
I've now deleted that branch on my local repo as well as on the origin.

Need to figure a way to clean up and update the pull request...

On Tue, Oct 24, 2017 at 2:30 AM, Ian Webster <notifications@github.com>
wrote:

> Nice job for your first time. The build is probably failing because you
> forked an old version of the repo. Maybe if you synced master (git
> checkout master and git pull), merged it into your branch (git checkout
> hello-devwahala and git merge master), and updated this pull request (git
> push), the build will pass.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/StayWokeOrg/general-congress-hotline/pull/51#issuecomment-338753937>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AfcrF1k9tK6Oi3JeodAQXXsQqvMtba9eks5svNtMgaJpZM4QDO2o>
> .
>
